### PR TITLE
Fix/pdb highvoltage net

### PR DESCRIPTION
**Pull request by TimBuckers**

### CODE REVIEW

File: march_hardware_interface/src/march_hardware_interface.cpp [hunk 1]
```diff
@@ -181,6 +185,7 @@ void MarchHardwareInterface::init()
           throw std::runtime_error("Joint has no high voltage net number");
         }
       }
+      //
```
Comment (TimBuckers) [inline march_hardware_interface/src/march_hardware_interface.cpp]:
```suggestion
```

**Review by martijnvandermarel — APPROVED** [review]
noice

File: march_hardware_interface/src/march_hardware_interface.cpp [hunk 2]
```diff
@@ -117,7 +117,11 @@ void MarchHardwareInterface::init()
         errorStream << "Joint " << joint_names_[i].c_str() << " has no net number";
         throw std::runtime_error(errorStream.str());
       }
-      marchRobot.getPowerDistributionBoard()->getHighVoltage().setNetOnOff(false, netNumber);
+      while(!marchRobot.getPowerDistributionBoard()->getHighVoltage().getNetOperational(netNumber)){
```
Comment (Ishadijcks) [inline march_hardware_interface/src/march_hardware_interface.cpp]: Style issues

File: march_hardware_interface/src/march_hardware_interface.cpp [hunk 3]
```diff
@@ -117,7 +117,11 @@ void MarchHardwareInterface::init()
...
+        marchRobot.getPowerDistributionBoard()->getHighVoltage().setNetOnOff(true, netNumber);
+        usleep(100000);
```
Comment (Ishadijcks) [inline march_hardware_interface/src/march_hardware_interface.cpp:123]: Add an explanation here

File: march_pdb_state_controller/src/march_pdb_state_controller.cpp [hunk 4]
```diff
@@ -50,13 +46,9 @@ void MarchPdbStateController::turnLowVoltageNetOnOrOffCallBack(const std_msgs::I
 bool MarchPdbStateController::init(march_hardware_interface::MarchPdbStateInterface* hw, ros::NodeHandle& root_nh,
                                    ros::NodeHandle& controller_nh)
 {
-  ROS_INFO("init MarchPdbStateController");
   // Get all temperature_sensors from the hardware interface
   const std::vector<std::string>& pdb_state_names = hw->getNames();
   for (unsigned i = 0; i < pdb_state_names.size(); i++)
-    ROS_INFO("Got pdb state %s", pdb_state_names[i].c_str());
-
-  ROS_INFO("amount of pdb_state_names: %zu", pdb_state_names.size());
```
Comment (Ishadijcks) [inline march_pdb_state_controller/src/march_pdb_state_controller.cpp:59]: Should these all be removed or rewritten to debugs?

**Review by Ishadijcks — CHANGES_REQUESTED** [review]
Needs some changes

File: march_pdb_state_controller/src/march_pdb_state_controller.cpp (hunk 4 above)
Comment (TimBuckers) [reply march_pdb_state_controller/src/march_pdb_state_controller.cpp:59]: I don't think the logs are useful anymore, most of the prints were used to see how far the code would run

File: march_hardware_interface/src/march_hardware_interface.cpp (hunk 3 above)
Comment (TimBuckers) [reply march_hardware_interface/src/march_hardware_interface.cpp:123]: Why do you think this code is not readable/understandable?

Comment (Ishadijcks) [reply march_hardware_interface/src/march_hardware_interface.cpp:123]: Ah nevermind, misread the sleep location

Comment (Ishadijcks) [reply march_hardware_interface/src/march_hardware_interface.cpp:123]: Add a timeout here though?

Comment (TimBuckers) [reply march_hardware_interface/src/march_hardware_interface.cpp:123]: Not needed

**Review by Ishadijcks — APPROVED** [review]

**Pull request merged**